### DISCUSSION TITLE
Resizable terminal (log + CLI) and compose editor

### DISCUSSION THREAD

**Discussion by jicho** [louislam/dockge#548] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

It would be nice when it would be possible to be able to resize / "blow up" the Terminal log overview, the Terminal CLI thingie and the compose viewer/editor.

On some smaller screens there is a lot of wrapping of lines, when it would be possible to add an "expand" option that puts the "widget" on top of the others the problem is solved and you can switch back easily.

### ✔️ Solution

Add an expand icon:

<img width="575" alt="image" src="https://github.com/louislam/dockge/assets/610740/d1a7020b-1a06-41ad-abfb-48de5421e58e">

Or make it possible to toggle the width of other elements.

**Comment by mediocreasian** [comment]
Adding this, until now there's no option to allow resizing of terminal

**Comment by Asinin3** [comment]
I just want to see the left column with all your docker images not be so wide, it takes up so  much space an makes the console and yaml editor wrap lines way too easily. Resizable would be nice but really just make the side-bar not as wide and it would be so much better and much easier to implement..

<img width="3840" height="2048" alt="image" src="https://github.com/user-attachments/assets/9d0c0de7-b18a-4215-9973-beb36e343032" />

**Comment by JaredT6694** [comment]
I'd also like to see the update terminal output dynamically resize, maybe to the number of containers.  I have some compose files with 10+ containers, the update only displays maybe 7 at a time.
